package.json

```diff
@@ -43,11 +43,11 @@
     "truffle-plugin-verify": "^0.4.0"
   },
   "dependencies": {
-    "@gnosis.pm/dex-contracts": "^0.4.2",
+    "@gnosis.pm/dex-contracts": "^0.4.3",
     "@gnosis.pm/owl-token": "^3.1.0",
-    "@gnosis.pm/safe-contracts": "v1.1.1-dev.2",
+    "@gnosis.pm/safe-contracts": "v1.2.0",
```

Comment:
I already did the research and I believe that this version switch is safe in this case.
On the other end, I think the change deserves its own PR, and without any check it's very risky because the code in this repo would use the artifacts of a Safe v1.2.0 to manage the currently existing Safe v1.1.1, and there are no unit tests covering this scenario.